test/unit/vclock.cc

```diff
@@ -444,17 +444,82 @@ test_minmax(void)
 
 #undef test
 
+#define test(s, n, expected) ({						\
+	struct vclock vc;						\
+	vclock_create(&vc);						\
+	vclock_from_string(&vc, (s));					\
+	is(vclock_nth_element(&vc, n), expected,			\
+		"vclock_nth_element \"%s\", %d => %d", s, n, expected);	\
+})
+
+int
+test_nth_element(void)
```

Comment:
Please, make this and the other function `static`.

Reply:
Fixed.